common/build.gradle

```diff
@@ -349,3 +349,8 @@ tasks.configureEach { task ->
         task.dependsOn 'sourcesJar'
     }
 }
+
+// This is used to generate the pom file for publishing to external maven in maven-release-jobs.yml
+tasks.withType(GenerateMavenPom).all {
+    destination = layout.buildDirectory.file("poms/${project.name}-${project.version}.pom").get().asFile
+}
```

Comment:
is this a new requirement for ESRP? We published pom.xml files earlier but it looks like we would need .pom files now?

Reply:
yes.